martini.go

```diff
@@ -57,6 +57,12 @@ func (m *Martini) Action(handler Handler) {
 	m.action = handler
 }
 
+// Action sets the logger
```

Comment:
Ok, fair enough, just replace the `Action` with `Logger` in the comment and LGTM